axelrod/tournament.py

```diff
@@ -101,7 +101,7 @@ def play(self, build_results=True, filename=None,
             self.progress_bar = tqdm.tqdm(total=len(self.match_generator),
                                           desc="Playing matches")
 
-        if on_windows and (filename is None):
+        if on_windows and (filename is None):  # pragma: no cover
```

Comment:
I think this is the best way. There is no function to make the skip pragma conditional, so we just need to pick whether coverage is called from travis or appveyor. I reckon we stick as we currently have it and use the pragma to skip windows stuff.